cgmanifest.json

```diff
@@ -18279,6 +18320,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "perl-XML-SAX",
+          "version": "1.02",
+          "downloadUrl": "http://www.cpan.org/authors/id/G/GR/GRANTM/XML-SAX-1.02.tar.gz"
```

Comment:
We're using a modified source tarball, not the one from this URL.

Reply:
Correct. It's updated.